kafka/consumer_test.go

```diff
@@ -176,10 +179,12 @@ func TestConsumerAPIs(t *testing.T) {
 	// OffsetsForTimes
 	offsets, err := c.OffsetsForTimes([]TopicPartition{{Topic: &topic, Offset: 12345}}, 100)
 	t.Logf("OffsetsForTimes() returned Offsets %s and error %s\n", offsets, err)
-	if err == nil {
+	if !c.IsClosed() && err == nil {
 		t.Errorf("OffsetsForTimes() should have failed\n")
+	} else if c.IsClosed() && err != errCheck {
+		t.Errorf("OffsetsForTimes() should have thrown %v but throwed %v", errCheck, err)
```

Comment:
"threw"

Reply:
nit: 
thrown -> threw

Reply:
nit:
throwed->threw, not thrown to threw. Or so I hope :sweat_smile: